src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -4026,6 +4027,17 @@ public void NullableEnable()
             VerifyDirectivesSpecial(node, new DirectiveInfo { Kind = SyntaxKind.NullableDirectiveTrivia, Status = NodeStatus.IsActive, Text = "enable" });
         }
 
+        [Fact]
+        [Trait("Feature", "Directives")]
+        public void NullableSafeOnly()
+        {
+            var text = @"#nullable safeonly";
+            var node = Parse(text, options: TestOptions.Regular8);
+            TestRoundTripping(node, text);
+            VerifyErrorCode(node); // no errors
+            VerifyDirectivesSpecial(node, new DirectiveInfo { Kind = SyntaxKind.NullableDirectiveTrivia, Status = NodeStatus.IsActive, Text = "safeonly" });
+        }
```

Comment:
>} [](start = 8, length = 1)

Do you want to update AllInOneCSharpCode too? (the test/resource with every syntax)

Reply:
> I think it'll be faster to update that test than to file an issue (I don't think we have one) ;-)

Opened #31882. I am pretty sure that was faster.

---
In reply to: [242356765](https://github.com/dotnet/roslyn/pull/31806#discussion_r242356765) [](ancestors = 242356765)